tests/foreman/ui/test_subscription.py

```diff
@@ -121,14 +115,11 @@ def test_positive_end_to_end(session, target_sat):
             ignore_error_messages=['Danger alert: Katello::Errors::UpstreamConsumerNotFound'],
         )
         assert session.subscription.has_manifest
-        # dashboard check
-        subscription_values = session.dashboard.read('SubscriptionStatus')['subscriptions']
-        assert subscription_values[0]['Subscription Status'] == 'Active Subscriptions'
-        assert int(subscription_values[0]['Count']) >= 1
-        assert subscription_values[1]['Subscription Status'] == 'Subscriptions Expiring in 120 Days'
-        assert int(subscription_values[1]['Count']) == 0
-        assert subscription_values[2]['Subscription Status'] == 'Recently Expired Subscriptions'
-        assert int(subscription_values[2]['Count']) == 0
+        subscriptions = session.subscription.read_subscriptions()
+        assert len(subscriptions) >= 1
+        assert 'Red Hat' in subscriptions[0]['Name']
```

Comment:
```suggestion
        assert any('Red Hat' in sub['Name'] for sub in subscriptions)
```
Only thought is perhaps we could assert what ~any entry Name contains, not just the first entry `subscriptions[0]`. In case this manifest changes at some point to include non-Red Hat subs?
Otherwise, looks good and passes CI, could be a non-blocking enhancement

Reply:
cool @damoore044 suggested changes working as expected